src/BulletSoftBody/btSoftBodyHelpers.h

```diff
@@ -140,6 +142,9 @@ struct btSoftBodyHelpers
 											bool bfacelinks,
 											bool btetralinks,
 											bool bfacesfromtetras);
+    static btSoftBody* CreateFromVtkFile(btSoftBodyWorldInfo& worldInfo, const char* vtk_file);
+    static btSoftBody* CreateFromVtkFile(btSoftBodyWorldInfo& worldInfo, std::istream& in);
```

Comment:
remove version of CreateFromVtkFile with std::isteam, and only use const char* vtk_file);?

Reply:
removed the std::istream version.